### ISSUE TITLE
Can't import from module 're.translate'

### ISSUE THREAD

**Issue by notzoom** [QQuick/Transcrypt#443]
Error while compiling (offending file last):
	File '/home/user/Dropbox/Python/templating.py', line 1, at import of:
	File '/usr/local/lib/python3.5/dist-packages/transcrypt/modules/re/__init__.py', line 15, at import of:
	File '/usr/local/lib/python3.5/dist-packages/transcrypt/modules/re/translate.py', line 15, namely:
	
	Can't import from module 're.translate'

**Comment by JdeH** [comment]
The re module isn't fully complete yet.
Also note that Transcrypt cannot import any modules written in C, since that isn't supported in the browser.
You importing a module that imports C modules will fail.

**Comment by notzoom** [comment]
@JdeH What is the status of the re module ? Could a donation directed towards it help get it done faster and what would that amount have to be like ? Thanks.

**Comment by JdeH** [comment]
@notzoom 

The re module was contributed by @RCL-Carl and @GULPF.
I guess they get notified if they are mentioned here.
The most appropriate thing probably would be if one or both of them would finish this module, and receive any benefits connected to that.

I do not know how to explictly notify @GULPF, but I've assigned this to @RCL-Carl.
@RCL-Carl: are you interested in finishing this, maybe for a fee?
Feel free not to, of course.
It's you two have the credits for creating this module in the first place.

Meanwhile I'll also take a look at the guts of re to get a rough idea of what would be needed.

**Comment by GULPF** [comment]
If @RCL-Carl is not available I'm willing to finish the re module, I'll probably have time for it in 2-3 weeks.

**Comment by notzoom** [comment]
@GULPF Hello there ! Feel free to let us know once you have updates.

**Comment by JdeH** [comment]
@GULPF 
I am curious what the status of this issue is.
Do you still plan to work on this?
An progress yet?

**Comment by GULPF** [comment]
Yes, sorry for the delay. I'll open a PR with my progress later today.

**Comment by JdeH** [comment]
No problem.
Just very glad you're working on this!
